plasma/plasma-spec.ini

```diff
@@ -77,6 +77,45 @@ requires:
     andBool 0 <=Int TOKEN_ID     andBool TOKEN_ID <Int (2 ^Int 160)
     andBool 0 <=Int EXITS_QUEUE_ID    andBool EXITS_QUEUE_ID <Int (2 ^Int 160)
 
+[submitBlock]
+output: _ => _
+callData: #abiCallData("submitBlock", #bytes32(ROOT_HASH))
+gas: {GASCAP} => _
+callValue: _
+refund: _
+requires:
+    andBool #rangeAddress(OPERATOR)
+    andBool #rangeBytes(32, ROOT_HASH)
+    
+[submitBlock-success]
+k: #execute => #halt
+statusCode: _ => EVMC_SUCCESS
+log: _:List ( .List => ListItem(#abiEventLog(ACCT_ID, "BlockSubmitted", #uint256(CURRENTCHILDBLOCK))) )
+storage:
+    #hashedLocation({COMPILER}, {_OPERATOR}, .IntList) |-> OPERATOR
+    #hashedLocation({COMPILER}, {_CURRENTDEPOSITBLOCK}, .IntList) |-> (CURRENTDEPOSITBLOCK => 1)
+    #hashedLocation({COMPILER}, {_CURRENTCHILDBLOCK}, .IntList) |-> (CURRENTCHILDBLOCK => CURRENTCHILDBLOCK +Int 1000)
+    #hashedLocation({COMPILER}, {_CHILDCHAIN}, CURRENTCHILDBLOCK) +Int 0 |-> (ROOT => ROOT_HASH)
+    #hashedLocation({COMPILER}, {_CHILDCHAIN}, CURRENTCHILDBLOCK) +Int 1 |-> (CHILD_TIME_STAMP => TIME_STAMP)
+    _:Map
++requires:
+    andBool OPERATOR ==Int CALLER_ID
+    andBool 0 <=Int CURRENTDEPOSITBLOCK
+    andBool CURRENTDEPOSITBLOCK +Int 1 <Int 1000
+    andBool 1000 <=Int CURRENTCHILDBLOCK andBool (CURRENTCHILDBLOCK +Int 1000) <Int (2 ^Int 256)
+    andBool #rangeBytes(32, ROOT_HASH)
```

Comment:
duplicated with L88 (https://github.com/LayerXcom/verified-smart-contracts/pull/25/files#diff-7cb27c3b7bb4f3f18e5a34b9a681d8aaR88 )?

Reply:
ah, this `ROOT_HASH` means `ROOT`?